docs/commands/ts.mrange.md

```diff
@@ -8,7 +8,7 @@ TS.MRANGE fromTimestamp toTimestamp
           [FILTER_BY_VALUE min max]
           [WITHLABELS | SELECTED_LABELS label...]
           [COUNT count]
-          [[ALIGN value] AGGREGATION aggregator bucketDuration [EMPTY]]
+          [[ALIGN value] AGGREGATION aggregator bucketDuration [BUCKETTIMESTAMP bt] [EMPTY]]
```

Comment:
@OfirMos can we include the relevant description of `BUCKETTIMESTAMP` in the command description.